.github/workflows/cpp_linter.yaml

```diff
@@ -14,12 +14,14 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
-          style: llvm
+          style: file
+          # An empty string means "use the .clang-tidy file"
+          tidy-checks: ""
           lines-changed-only: false
           files-changed-only: true
 
       - name: Fail Fast?!
         if: steps.linter.outputs.checks-failed > 0
         run: echo "Some files failed the linting checks!"
         # for actual deployment
-        # run: exit 1
\ No newline at end of file
+        # run: exit 1
```

Comment:
Is there still a blank line at the end of the file? I like to maintain a trailing '\n'

Reply:
Yup, there wasn't one before.